digdag-core/src/main/java/io/digdag/core/agent/CheckedConfig.java

```diff
@@ -9,50 +9,33 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import io.digdag.client.config.Config;
 
-public class CheckedConfig
```

Comment:
previous revert commit was separating this PR from #343 but later I found that this PR actually depends on #343.